src/lib/capacityEngine.js

```diff
@@ -235,10 +199,11 @@ function getPoolCapacity(store, poolId, { periodSeconds = SECONDS_PER_DAY.mul(30
     .map(productId =>
       calculateProductCapacity(store, productId, {
         poolId,
-        periodSeconds,
+        period,
         now,
         assets,
         assetRates,
+        withPools: false,
```

Comment:
since all endpoints share this generic `calculateProductCapacity` function we had to keep `withPools` param _internally_ as `capacityPerPools` does not make sense to have on the other endpoints